### ISSUE TITLE
Undefined function: SpaceVim#logger#warn

### ISSUE THREAD

**Issue by ronisbr** [wsdjeg/dein-ui.vim#8]
Sometimes when I run `:DeinUpdate` and there is an error or warning, I see this message:

```
Erro detectado ao processar function <SNR>129_on_pull_exit[45]..<SNR>129_recache_rtp[7]..<SNR>129_reinstall_update_failed:
linha    4:
E117: Função desconhecida: SpaceVim#logger#warn
Erro detectado ao processar function <SNR>129_on_pull_exit[45]..<SNR>129_recache_rtp[7]..<SNR>129_reinstall_update_failed:
linha    4:
E117: Função desconhecida: SpaceVim#logger#warn
```

The last line is saying: `Undefined function: SpaceVim#logger#warn`.

**Comment by wsdjeg** [comment]
@ronisbr please use issue tempalte, debug info is needed.

**Comment by ronisbr** [comment]
@wsdjeg where can I find the template? When I created the issue, it was blank.

**Comment by wsdjeg** [comment]
Ok, it seems dein-ui missing some files form spacevim, I will fix it.

**Comment by Chiarandini** [comment]
Hey. I'm getting the same problem with "Unknown function: SpaceVim#logger#warn"

**Comment by TamaMcGlinn** [comment]
I'm getting this error as well. I noted that the latest commits on the master branch were made by 'SpaceVimBot' - perhaps SpaceVim is required for this plugin now?

**Comment by TamaMcGlinn** [comment]
Found a fix; replace `wsdjeg/dein-ui` plugin with the fork `Freed-Wu/dein-ui`.

**Comment by TamaMcGlinn** [comment]
I'm not sure how detach_plugin.sh is supposed to work, but I'm now getting errors of the form:

```
Key not present in Dictionary: "set_name"
```

And also for set_level, set_silent, set_verbose, get_name() and warn

**Comment by masa300V** [comment]
I have the same situation on my Windows 10 (Neovim 0.5.0, Windows-Git v2.33.0(2), Windows-Python 3.9.7) where having the error of "set_name" when I run    :DeinUpdate  .  After running the dein-ui, I also noticed that many zombie "git.exe" processes running (I think the same number of zombies as my plugin list).

I found this after small checking: 
By copying below file into "autoload/SpaceVim/api folder of dein's cache, no more "set_name" error but now it trys to download something missing and stack there (no clean finish).
  https://raw.githubusercontent.com/SpaceVim/SpaceVim/master/autoload/SpaceVim/api/logger.vim
My guess is we need some more files from SpaceVim into this dein-ui distribution package under the autoload.

On my case, this no "set_name" error started these weeks, very recently, it was working until July-Aug 2021.
